marshal

```diff
@@ -6,9 +6,6 @@ import logging
 import wlutil
 import contextlib
 
-if 'RISCV' not in os.environ:
```

Comment:
You don't depend on this variable anymore?

Reply:
We never depended on it explicitly. I was just using it as a proxy to detect if you'd sourced sourcme. In reality, we just depend on the riscv toolchain. I just added back a check for that instead.